src/javascripts/components/back-to-top.mjs

```diff
@@ -3,7 +3,7 @@ class BackToTop {
    * @param {Element} $module - HTML element
    */
   constructor ($module) {
-    if (!($module instanceof HTMLElement)) {
+    if (!($module instanceof HTMLElement) || !document.body.classList.contains('govuk-frontend-supported')) {
```

Comment:
All these identical constructor-y bits just make me want to do a base component sooner!